pyqtgraph/graphicsItems/GridItem.py

```diff
@@ -26,6 +26,30 @@ def __init__(self, pen='default', textPen='default'):
         self.setPen(pen)
         self.setTextPen(textPen)
         self.setTickSpacing(x=[None, None, None], y=[None, None, None])
+        self.axes = None
+        self.ctrl = None
+
+
+    def setAxes(self, axes):
+        """ 
+        Set the axes object, in case the GridItem belongs to plot, whose axes are controlled by AxisItem's
+        """
+        self.axes = axes
+
+
+    def setCtrl(self, ctrl):
```

Comment:
nit-pick suggestion, how do you feel about renaming to `setControl`; I've been trying to avoid using abbreviations for methods/variables/functions in the public API.

Reply:
Also will need a getter method to get the content of the `ctrl` attribute

Reply:
Method name `setCtrl` Changed to `setControl` and added the getter method for the `ctrl` attribute with commit e88d5ab